src/libs/state.ts

```diff
@@ -145,6 +145,8 @@ export const contactUsActive = atom(false);
 
 export const workspaceStore = atom<any>(undefined);
 
+export const workspaceCheckCloudAccessFailureStore = atom<number>(0);
```

Comment:
Doesn't this mean the same store will be used for all workspaces? 

I have to wonder again… is this change really necessary?

Reply:
Its not necessary, and been removed.